### PR TITLE
add a test that shared library can be loaded + make shared library loadable on linux :)

### PR DESCRIPTION
**Pull request by djsutherland**
Test for the problem of #5.

### CODE REVIEW

**Comment by djsutherland** [comment]
The CircleCI failure seems to be a legitimate failure of the test:
```
libquadmath.so.0: cannot open shared object file: No such file or directory
```

**Comment by jschueller** [comment]
seems libgcc is not pulled but libgfortran after all

**Comment by jschueller** [comment]
I think libquadmath.so should be part of libgfortran rather than libgcc because libgfortran.so links to it, and non of the libs in libgcc link to it: libstdc++.so, libgccc_s.so, libgomp.so 
what do you think @jakirkham

**Comment by jakirkham** [comment]
So the Linux `libgfortran` is coming from `defaults`. I have been reluctant to package our own copy at `conda-forge` (as we do on macOS), but maybe it is time we did.

**Comment by jschueller** [comment]
@dougalsutherland could you add ligcc to run requirements ?

**Comment by djsutherland** [comment]
@jschueller Done, and passing on CircleCI now.

I guess the libgcc requirement isn't strictly necessary on Mac, but I didn't selector it anyway....

**Comment by jschueller** [comment]
could you also bump the build number ?

**Comment by jschueller** [comment]
I like this, this is a good way of testing lib-only packages, thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,14 +26,17 @@ requirements:
 
   run:
     - libgfortran
+    - libgcc
```
Comment (jakirkham) [inline recipe/meta.yaml]: Would recommend doing the following if we are adding `libgcc`.

```yaml
    - libgfortran  # [osx]
    - libgcc       # [linux]
```

Unfortunately it has generally been a pain to have `libgcc` as a `run` dependency on macOS. Particularly for low level packages like `matplotlib`, `numpy`, and `scipy` as examples.

Comment (djsutherland) [reply recipe/meta.yaml]: Sure.

**Pull request merged**